docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1506,6 +1506,13 @@ The estimated number of operations to use the trained model.
 `license_level`:::
 (string)
 The license level of the trained model.
+
+`default_field_mappings` :::
+(object)
+A string to string mapping that contains the default field mappings to use
+when inferring against the model. Any field mapping described in the
```

Comment:
Should we add a line here about multi fields which is the primary reason for this change?

Reply:
For sure!